packages/shylu/shylu_node/tacho/unit-test/Tacho_TestDenseLinearAlgebra.hpp

```diff
@@ -82,44 +66,35 @@ TEST( DenseLinearAlgebra, team_gemm_nn ) {
   B1.modify_device();
   C1.modify_device();
 
-  Kokkos::Random_XorShift64_Pool<typename DeviceType::execution_space> random(13718);
+  Kokkos::Random_XorShift64_Pool<typename device_type::execution_space> random(13718);
+
+  Kokkos::fill_random(A1.d_view, random, value_type(1));
+  Kokkos::fill_random(B1.d_view, random, value_type(1));
+  Kokkos::fill_random(C1.d_view, random, value_type(1));
 
-  Kokkos::fill_random(A1.d_view, random, ValueType(1));
-  Kokkos::fill_random(B1.d_view, random, ValueType(1));
-  Kokkos::fill_random(C1.d_view, random, ValueType(1));
-  
   Kokkos::deep_copy(A2.h_view, A1.d_view);
   Kokkos::deep_copy(B2.h_view, B1.d_view);
   Kokkos::deep_copy(C2.h_view, C1.d_view);
-  
+
   // tacho test
-  ::Test::Functor_TeamGemm test(transa, transb, 
-                                m, n, k,
-                                alpha, A1, B1, beta, C1);
+  ::Test::Functor_TeamGemm test(transa, transb, m, n, k, alpha, A1, B1, beta, C1);
   test.run();
 
-  // reference test 
-  Blas<ValueType>::gemm(transa, transb, 
-                        m, n, k,
-                        alpha, 
-                        A2.h_view.data(), A2.h_view.stride_1(),
-                        B2.h_view.data(), B2.h_view.stride_1(),
-                        beta,
-                        C2.h_view.data(), C2.h_view.stride_1());
-  
-  const MagnitudeType eps = std::numeric_limits<MagnitudeType>::epsilon() * 1000;
-  for (int i=0;i<m;++i)
-    for (int j=0;j<n;++j) 
-      EXPECT_NEAR(abs(C1.h_view(i,j)), abs(C2.h_view(i,j)), eps);
-  TEST_END;
+  // reference test
+  Blas<value_type>::gemm(transa, transb, m, n, k, alpha, A2.h_view.data(), A2.h_view.stride_1(), B2.h_view.data(),
+                         B2.h_view.stride_1(), beta, C2.h_view.data(), C2.h_view.stride_1());
+
+  const magnitude_type eps = std::numeric_limits<magnitude_type>::epsilon() * 1000;
+  for (int i = 0; i < m; ++i)
+    for (int j = 0; j < n; ++j)
+      EXPECT_NEAR(ats::abs(C1.h_view(i, j)), ats::abs(C2.h_view(i, j)), eps);
```

Comment:
@e10harvey check this.

Reply:
now this pr passed. I just merge this first.

Reply:
@kyungjoo-kim: I understand where you're coming from. However, let's focus on getting master merge working again before introducing additional changes & features that could result in new failures that block master merge. Please move this fix into #10625.

Reply:
@kyungjoo-kim: Getting the master branch updated is a top-priority for the product leaders. I will make sure this PR is next in the priority queue once the master branch is updated.